IntelliTect.TestTools.Console/ConsoleAssert.cs

```diff
@@ -67,14 +122,14 @@ public static string Expect(string expected, Action<string[]> action, bool norma
         public static void Expect<T>(string expected, Func<string[], T> func, T expectedReturn = default, params string[] args)
         {
             T @return = default;
-            Expect(expected, () => @return = func(args));
+            Expect(expected, () => @return = func(args), NormalizeOptions.None);
```

Comment:
Why default to `NormalizeOptions.None`?  Shouldn't we have both normalize line endings and strip ansi escape code to true?  (In other words `NormalizeOptions.Default` once we have it?)

Reply:
Yes! Thank you